### ISSUE TITLE
Show amount of users on delegations page

### ISSUE THREAD

**Issue by nadjaweisskopf** [leihs/leihs#954]
https://tom.leihs.zhdk.ch/admin/delegations/5e0c1811-0891-4541-865e-464c2f7c28ea

Can we show the amount of users added to this group on the third line? Just an Info?

**Comment by DrTom** [comment]
@nadjaweisskopf  yes number of groups and number of users can be shown here. It needs some effort but not too much.

**Comment by DrTom** [comment]
Remark: show prominently in the table not in the text below.
